From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bastien Montagne <bastien@blender.org>
Date: Tue, 17 Sep 2024 19:20:01 +0200
Subject: [PATCH] Fix warnings when building Blender with mold linker on linux.

Remove the three problematic entries:
- `aligned_free` is not a 'real' function and does not need to be
  handled here. There is no symetry with the `aligned_malloc` symbol
  here.
- `__end` and `_bss_start` seem to have been mis-typed from the
  beginning (actual symbols seem to be `_end` and `__bss_start`).
  Regardless, neither `gold` nor `mold` seem to take these directives
  from the LD version script into account (result is the same for them
  when removed for the `.map` file), so removing them.

Pull Request: https://projects.blender.org/blender/blender/pulls/127761
---
 source/creator/symbols_unix.map | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/source/creator/symbols_unix.map b/source/creator/symbols_unix.map
index 8500c9fcd34..09b191a8acd 100644
--- a/source/creator/symbols_unix.map
+++ b/source/creator/symbols_unix.map
@@ -5,8 +5,6 @@
 {
 global:
   /* Essential symbols for the program to start and exit. */
-  _bss_start;
-  __end;
   _fini;
   _init;
   /* Needed for Python modules to work. */
@@ -24,7 +22,6 @@ global:
   __Zdl*;
   __Zda*;
   aligned_alloc;
-  aligned_free;
   calloc*;
   free;
   mallinfo;
